yomi/simple_eval.py

```diff
@@ -0,0 +1,42 @@
+""" Simple job that reads from a storage bucket """
+
+import os
+import apiclient.discovery
+import httplib2
+import json
+from oauth2client.service_account import ServiceAccountCredentials
+
+SERVICE_ACCOUNT_KEY_LOCATION = os.environ['SERVICE_ACCOUNT_KEY_LOCATION']
+BUCKET_NAME = os.environ['BUCKET_NAME']
+
+
+def run():
+    """ Talk to GCS. Do some things. """
```

Comment:
it does indeed look like some things are being done!

Reply:
very few, as of yet